rpm/0003-SSO-integration.patch

```diff
@@ -3763,3 +3758,6 @@ index 5807dc01..3a79d840 100644
  SRCDIR = $$PWD/../../src/plugins/messageservices/smtp
  INCLUDEPATH += $$SRCDIR
  QT += qmfclient qmfmessageserver
+-- 
+2.25.1
```

Comment:
Rather not have these, git format-patch [...] --no-signature.

Reply:
Ah, I forgot this one, sorry. Fixed.